#############################################################
#                                                           #                                                        #
# Authors: zhang cheng                    #

#############################################################

dlo:

  version: 1.0.0

  adaptiveParams: false

  imu: true
  gravityAlign: false

  odomNode:
    odom_frame: odom
    child_frame: base_link

  mapNode:
    publishFullMap: true
    publishFreq: 1.0
    leafSize: 0.25
